packages/doenetml-worker-rust/lib-doenetml-core/src/core/state/state_var.rs

```diff
@@ -535,17 +559,66 @@ impl<T: Default + Clone> StateVarReadOnlyView<T> {
     }
 }
 
+#[enum_dispatch]
+pub trait QueryUpdateRequests {
+    /// Reset variable that tracks whether or not an update has been requested
```

Comment:
The implementation doesn't look like it is resetting anything, since it's just setting it to false...Maybe `clear_queued_update` would be a better name?